tests/diff.yml

```diff
@@ -5,12 +5,14 @@ args: ["-timeout=20s", "-shuffle=on", "-race", "./..."]
 results:
   ferretdb:
     stats:
-      expected_fail: 9
-      expected_pass: 4
+      expected_fail: 14
+      expected_pass: 6
 
     pass:
       - github.com/FerretDB/dance/tests/diff/TestCollectionName/Length200/FerretDB
```

Comment:
@noisersup can this be replaced with something like
```
- `regex`: /dance/tests/diff/.+/FerretDB$
```
?

Reply:
It should work let me try

Reply:
> @noisersup can this be replaced with something like
> 
> ```
> - regex: /dance/tests/diff/.+/FerretDB$
> ```
> 
> ?

it can be done in a separate PR.

Reply:
It can be, but let's do that in this PR if it works – that will make diff configuration more clean

Reply:
I will do that in #177 then :)

Reply:
> It can be, but let's do that in this PR if it works – that will make diff configuration more clean

Test failed when @noisersup pushed.